src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/notifications/Notification.java

```diff
@@ -0,0 +1,30 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.notifications;
+
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import lombok.Builder;
+import lombok.Getter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.support.FindBy'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)